plugins/rbac-backend/src/__fixtures__/data/valid-csv/rbac-policy.csv

```diff
@@ -0,0 +1,10 @@
+g, user:default/guest, role:default/catalog-writer
+g, user:default/guest, role:default/catalog-reader
+g, user:default/guest, role:default/catalog-deleter
+
+p, role:default/catalog-writer, catalog-entity, update, allow
+p, role:default/catalog-writer, catalog-entity, read, allow
+p, role:default/catalog-writer, catalog.entity.create, use, allow
+p, role:default/catalog-deleter, catalog-entity, delete, deny
+
+p, user:default/known_user, test.resource.deny, use, allow
```

Comment:
Maybe we should not promote direct assign permission to the role in the tests.

Reply:
Yes, I agree